### PR TITLE
docs: add http-libs and type-fest

### PR DESCRIPTION
**Pull request by jeroenptrs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ Consider submitting to [the deno.land/x](https://github.com/denoland/registry/bl
 - [expressive](https://github.com/jinjor/deno-playground/tree/master/expressive) - An express-like web app framework.
 - [flags](https://github.com/denoland/deno_std/tree/master/flags) - Command line arguments parser for Deno based on minimist.
 - [http](https://github.com/denoland/deno_std/tree/master/http) - HTTP module including a file server.
+- [http-libs](https://github.com/denoserverless/http-libs) - HTTP modules and typings for Deno
```
Comment (kevinkassimo) [inline README.md]: nit: Remove `for Deno` and add trailing `.`

File: README.md [hunk 2]
```diff
@@ -86,6 +87,7 @@ Consider submitting to [the deno.land/x](https://github.com/denoland/registry/bl
 - [servest](https://github.com/keroxp/servest) - A progressive HTTP server/router.
 - [sql-builder](https://github.com/manyuanrong/sql-builder) - An sql query builder.
 - [textproto](https://github.com/denoland/deno_std/tree/master/textproto)
+- [type-fest](https://github.com/denoserverless/type-fest) - A collection of essential TypeScript types (Deno port of sindresorhus/type-fest)
```
Comment (kevinkassimo) [inline README.md]: ditto

File: README.md (hunk 1 above)
Comment (jeroenptrs) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (jeroenptrs) [reply README.md]: done

**Pull request merged**